airflow/plugins_manager.py

```diff
@@ -122,10 +122,11 @@ def is_valid_plugin(plugin_obj, existing_plugins):
     return False
 
 
-plugins_folder = configuration.conf.get('core', 'plugins_folder')
-if not plugins_folder:
-    plugins_folder = configuration.conf.get('core', 'airflow_home') + '/plugins'
-plugins_folder = os.path.expanduser(plugins_folder)
+plugins_folder = configuration.conf.get(
```

Comment:
Can we not add this line to settings.py and just use `plugins_folder=settings.plugins_folder` for uniformity?

Reply:
Good call, will do.